SEP-0002-revised.md

```diff
@@ -0,0 +1,244 @@
+
+For the first election of the coordinating committee, one member will have a one year term, one member a two year term and one member a three year term.
+The term length of the elected candidates may be decided between themselves, in the case agreement can not be reached the person elected to the financial officer role will serve three years, the deputy lead developer two years and the lead developer one year.
+
+### Removing Members
+
+In exceptional circumstances, the project members may remove one or more sitting steering committee members via a single vote.
+
+Such a vote is triggered when a project member calls for one publicly on an appropriate Project-wide communication channel, and two other active project members second the proposal.
+
+For Steering Committee Member(s) to be removed, two-thirds of the active Project Members (where the count includes the member being voted on if they are a project member) must vote in support of the removal.
+
+To avoid repeated removal votes, any individual Project Member who has called for, or seconded such a vote, may not call for or second a vote to remove any Steering Committee member for one year from the original motion.
+
+## The Project Members
+
+The project members are trusted community members with a proven track record of constructive contribution to the goals of the project.
+Collectively, the project members make decisions about the direction of the project and have authority over the project, which is delegated to the Steering Committee.
+
+Project Members are expected to act as role models for the community and custodians of the SunPy Project, on behalf of the community and all those who rely on SunPy.
+They will act as representatives of the SunPy Project, where necessary, including in online discussions or at official SunPy events.
+
+### Responsibilities and Rights
+
+Project Members may and should participate in formal votes on:
+
+* Adding new Project Members
+* Electing Steering Committee members
+* Appointment of the Ombudsperson
+* Removing Project Members
+* Removing Steering Committee members
+* Removing the Ombudsperson
+* Changes to this SEP
+* Other matters for which the Steering Committee believes a vote is appropriate.
+
+### Membership
+
+Project Members of the SunPy Project must demonstrate all of the following:
+
+* A good grasp of the philosophy of the SunPy Project
+* A solid track record of being constructive and helpful
+* Significant contributions to the Project's goals, in any form
+* A willingness to dedicate time to improving the Project
+* A willingness to recruit, train, and nominate new team members
+
+Project Membership acknowledges sustained and valuable efforts that align well with the philosophy and the goals of the SunPy Project.
+
+### Initial Membership
+
+To be able to elect new Project Members an initial set of Project Members are needed.
+People meeting the following criteria will be invited to form the initial group of Project Members.
```

Comment:
```suggestion
People meeting the following criteria will be invited to form the initial group of Project Members.
They should consider the above responsiblities and activate agree to taking them on if they choose to accept the invitation.
```
Or similar? I think it's important that the initial project members accept the same responsibliities new members are asked to take on.

Reply:
Great suggestion, this should definitely be explicit.